src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -2634,6 +2646,12 @@
     <PRIResource Include="$(MSBuildThisFileDirectory)UITestsStrings\en-US\Resources.resw" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Update="C:\Code\Massimo37\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml\MarkupExtensionTests\MarkupExtension.xaml.cs">
+      <DependentUpon>MarkupExtension.xaml</DependentUpon>
+    </Compile>
+    <Compile Update="C:\Code\Massimo37\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml\xBindTests\xBind.xaml.cs">
+      <DependentUpon>xBind.xaml</DependentUpon>
+    </Compile>
     <Compile Update="C:\Dev\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\BorderTests\Border_CornerOverlap.xaml.cs">
```

Comment:
Odd paths to your local repo :)

Reply:
@MatFillion  Whoa, why did it hard-code the path ??

Reply:
Fixed.